src/libs/SidebarUtils.ts

```diff
@@ -385,38 +395,14 @@ function getOptionData({
         } else {
             result.alternateText = lastMessageTextFromReport.length > 0 ? lastMessageText : ReportActionsUtils.getLastVisibleMessage(report.reportID, {}, lastAction)?.lastMessageText;
             if (!result.alternateText) {
-                result.alternateText = Localize.translate(preferredLocale, 'report.noActivityYet');
+                // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+                result.alternateText = ReportUtils.formatReportLastMessageText(getWelcomeMessage(report).concatMessage || Localize.translateLocal('report.noActivityYet'));
```

Comment:
Based on the above comment ^ Do we still have a case where we would return `report.noActivityYet`

Reply:
I just updated and we still have some case `messageText` is `undefined` like chat thread or task report.